doc/cookbook/source/references.bib

```diff
@@ -153,3 +153,15 @@ @article{Quinonero-Candela2005
  year = {2005},
  pages = {1939--1959},
 }
+@TECHREPORT{Wickelmaier03anintroduction,
+    author = {Florian Wickelmaier},
+    title = {An introduction to MDS},
```

Comment:
If you are referring to Bayesian Reasoning and Machine learning, there's no reference to MDS. Maybe Modern Multidimensional Scaling from Borg is ok? (also cited in the documentation of `CMultidimensionalScaling`)